recipes/wt/all/conanfile.py

```diff
@@ -90,106 +119,172 @@ def _configure_cmake(self):
+            self._cmake.definitions["MYSQL_FOUND"] = True
+        if self.options.get_safe("with_postgres"):
+            self._cmake.definitions["POSTGRES_LIBRARIES"] = ";".join(_gather_libs("libpq"))
+            self._cmake.definitions["POSTGRES_INCLUDE"] = ";".join(self.deps_cpp_info["libpq"].include_paths)
+            self._cmake.definitions["POSTGRES_FOUND"] = True
+        if self.options.get_safe("with_mssql") and self.settings.os != "Windows":
+            self._cmake.definitions["ODBC_LIBRARIES"] = ";".join(_gather_libs("odbc"))
+            self._cmake.definitions["ODBC_INCLUDE"] = ";".join(self.deps_cpp_info["odbc"].include_paths)
+            self._cmake.definitions["ODBC_FOUND"] = True
+        if self.settings.os == "Windows":
+            self._cmake.definitions["CONNECTOR_FCGI"] = False
+            self._cmake.definitions["CONNECTOR_ISAPI"] = self.options.connector_isapi
         else:
-            self._cmake.definitions['CONNECTOR_FCGI'] = self.options.connector_fcgi
-            self._cmake.definitions['CONNECTOR_ISAPI'] = False
+            self._cmake.definitions["CONNECTOR_FCGI"] = self.options.connector_fcgi
+            self._cmake.definitions["CONNECTOR_ISAPI"] = False
         self._cmake.configure(build_folder=self._build_subfolder)
         return self._cmake
 
     def build(self):
-        tools.replace_in_file(os.path.join(self._source_subfolder, 'CMakeLists.txt'), 'find_package(OpenSSL)', '#find_package(OpenSSL)')
-        tools.replace_in_file(os.path.join(self._source_subfolder, 'CMakeLists.txt'), 'INCLUDE(cmake/WtFindMysql.txt)', '#INCLUDE(cmake/WtFindMysql.txt)')
-        tools.replace_in_file(os.path.join(self._source_subfolder, 'CMakeLists.txt'), 'INCLUDE(cmake/WtFindPostgresql.txt)', '#INCLUDE(cmake/WtFindPostgresql.txt)')
+        self._validate_dependency_graph()
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "find_package(OpenSSL)", "#find_package(OpenSSL)")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "INCLUDE(cmake/WtFindMysql.txt)", "#INCLUDE(cmake/WtFindMysql.txt)")
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "INCLUDE(cmake/WtFindPostgresql.txt)", "#INCLUDE(cmake/WtFindPostgresql.txt)")
+        if self.settings.os != "Windows":
+            tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), "INCLUDE(cmake/WtFindOdbc.txt)", "#INCLUDE(cmake/WtFindOdbc.txt)")
         cmake = self._configure_cmake()
         cmake.build()
 
     def package(self):
         self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
         cmake = self._configure_cmake()
         cmake.install()
-        shutil.move(os.path.join(self.package_folder, "share", "Wt"), os.path.join(self.package_folder, "bin")) 
+        shutil.move(os.path.join(self.package_folder, "share", "Wt"), os.path.join(self.package_folder, "bin"))
```

Comment:
@ericLemanissier Why is it move to `bin` folder? Shouldn't it go to `res` folder?

Reply:
because I misinterpreted the hooks message `If you are trying to package a tool put all the contents under the 'bin' folder`

Reply:
Also:
- wt installs `wt_config.xml` in bin folder.
- There is a `WT_CONFIG_XML` env var which can be set.

@wdobbe Do you think that `wt_config.xml` and `Wt/resources` should go to a `res` folder, and `WT_CONFIG_XML` added as a env var in `package_info`?

Reply:
I think installing them in a res folder would make more sense, but either are fine with me.

Reply:
ok, so let's keep it as it is.